extensions/ql-vscode/src/extension.ts

```diff
@@ -319,10 +319,7 @@ async function activateWithInstalledDistribution(ctx: ExtensionContext, distribu
   ctx.subscriptions.push(commands.registerCommand('codeQL.quickQuery', async () => displayQuickQuery(ctx, cliServer, databaseUI)));
   ctx.subscriptions.push(commands.registerCommand('codeQL.restartQueryServer', async () => {
     await qs.restartQueryServer();
-    const response = await Window.showInformationMessage('CodeQL Query Server restarted.', 'Show Log');
-    if (response === 'Show Log') {
-      qs.showLog();
-    }
+    helpers.showAndLogInformationMessage('CodeQL Query Server restarted.');
```

Comment:
Sorry, my fault for not being clear; the helper just needs an extra (optional? defaulting to `logging.logger`?) `OutputChannelArgument` argument or something, since the log stream that this existing 'Show Log' button shows (which I think is what we want) is the one named 'CodeQL Query Server', rather than the default 'CodeQL Extension Log', which is what `helpers.ts` is getting from `logging.logger`.

Reply:
Darn. You're right. I wasn't reading carefully enough. I'll fix. Though, it's a little more complex since the last argument is varargs, but I should be able to do something.

Reply:
If we're making the wrappers around the underlying varargs methods on `vscode.window`, I wouldn't mind at all refactoring to make them take an explicit array argument instead.

Reply:
What about if it takes a `message` arg and an `options` arg. This will make it more flexible and future-proof since I'm sure we'll want to augment with different behaviour later.

Reply:
sure, sounds fine